### PR TITLE
add gender to user model

### PR DESCRIPTION
**Pull request by sjarvie**
I would like to start contributing to open source and think this is a great starting project. We could add in a gender field to the user for numerous reasons. If we are interested in BMI then we could bring in BMI charts(which are different depending on gender), and display a chart demonstrating where a given BMI/gender lies on that chart.

### CODE REVIEW

File: app/serializers/user_serializer.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 class UserSerializer < ActiveModel::Serializer
-  attributes :id, :email, :name, :height
+  attributes :id, :email, :name, :height. :gender
```
Comment (TigerWolf) [inline app/serializers/user_serializer.rb]: This should be a comma.

**Comment by jdjkelly** [comment]
Thanks for the contribution!

I think we should call this sex instead of gender, as that's probably the most accurate term. BMI varies because of obvious (and non-obvious) biological differences in muscular structure etc, which is what the term sex usually means, whereas gender usually means tends to mean something socially/culturally.

**Comment by sjarvie** [comment]
My original intention was to use gender for political correctness, but I agree with you completely. This app is about the quantification of physical aspects of the body, and by definition sex is a more fitting metric to qualify.

File: db/migrate/20140116212437_add_gender_to_users.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddGenderToUsers < ActiveRecord::Migration
```
Comment (TigerWolf) [inline db/migrate/20140116212437_add_gender_to_users.rb]: This file is no longer needed.

**Pull request closed without merging**